app/assets/stylesheets/application.scss

```diff
@@ -17,10 +17,10 @@
  @import "bootstrap-sprockets";
  @import "bootstrap";
 
- .container-fluid {
-   margin-left: 130px;
-   margin-right: 130px;
- }
+ // .container-fluid {
+ //   margin-left: 130px;
```

Comment:
Ahhhh!!! Don't put extra css in this file - it's set to sass now and will blow up. Make a different file! Happy to help set that up if it goes weird